lighthouse-plugin-publisher-ads/test/utils/resource-classification_test.js

```diff
@@ -163,4 +163,34 @@ describe('resource-classification', () => {
       });
     }
   });
+  describe('#isGptImplTag', () => {
+    const testCases = [
+      {
+        description: 'Standard tag matches',
+        url: new URL('https://securepubads.g.doubleclick.net/gpt/pubads_impl_19700101.js?1234'),
+        expectation: true,
+      },
+      {
+        description: 'Non-standard tag matches',
+        url: new URL('https://securepubads.g.doubleclick.net/gpt/pubads_impl_queso_19700101.js'),
```

Comment:
s/queso/modern/ for a real world example

Reply:
fine...